### ISSUE TITLE
Consolidate test commands

### ISSUE THREAD

**Issue by kimadactyl** [geeksforsocialchange/PlaceCal#1416]
## User story

As a developer I want my tests to run as fast as possible so stuff gets done faster. 

Currently we are running tests in two batches. This is slowing things down as one lot has to complete before the next can start and boot up the whole server.

However, switching to `rails test:all` is creating some new issue with port numbers. See attached.

## Acceptance criteria

- [ ] Current test.yml file just runs `rails test:all` instead of 2 sep commands

## Implementation notes & questions

<img width="532" alt="Screenshot 2022-08-01 at 15 37 40" src="https://user-images.githubusercontent.com/511734/182183909-637a12fa-a907-4827-86c8-2fbb8b357149.png">

## Implementation plan

_To be written by the developer_

**Comment by r-ferrier** [comment]
sounds good!
